<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiDialogFrame</class>
 <widget class="QDialog" name="uiDialogFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0">
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>269</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox2">
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="m_pPreview">
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <property name="spacing">
          <number>16</number>
         </property>
         <item row="0" column="0">
          <widget class="QFrame" name="frame4">
           <layout class="QGridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="textLabel1">
                <property name="text">
                 <string>Frame width</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="m_pSpinBoxWidth">
                <property name="value">
                 <number>4</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout">
              <item>
               <widget class="QPushButton" name="m_pButtonColor">
                <property name="text">
                 <string>Frame Color ...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="m_pFrameColor">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="textLabel3">
                <property name="text">
                 <string>Line Style</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="m_pComboStyle">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="textLabel3_2">
                <property name="text">
                 <string>Join Style</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="m_pComboJoin">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1" rowspan="4">
             <widget class="QGroupBox" name="m_pGroupGeometry">
              <layout class="QVBoxLayout">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <layout class="QGridLayout">
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="m_pEditY">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="textLabel2">
                   <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;X&lt;/p&gt;</string>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="textLabel2_2">
                   <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;Y&lt;/p&gt;</string>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="m_pEditX">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="spacer3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>21</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QGridLayout">
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="m_pEditHeight">
                   <property name="text">
                    <string>100</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="m_pEditWidth">
                   <property name="text">
                    <string>100</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="textLabel2_3">
                   <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;Width&lt;/p&gt;</string>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="textLabel2_4">
                   <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;Height&lt;/p&gt;</string>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="m_pFrameSelect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="m_pRadioImage">
          <property name="text">
           <string>Ima&amp;ge Frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pButtonImage">
          <property name="text">
           <string>Browse ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="m_pRadioPicture">
          <property name="text">
           <string>Pic&amp;ture Frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pButtonPicture">
          <property name="text">
           <string>Edit ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="m_pGroupAnimation">
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="m_pButtonAnimation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Animation
Script</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_pSpinBoxWidth</tabstop>
  <tabstop>m_pButtonColor</tabstop>
  <tabstop>m_pComboStyle</tabstop>
  <tabstop>m_pComboJoin</tabstop>
  <tabstop>m_pEditX</tabstop>
  <tabstop>m_pEditY</tabstop>
  <tabstop>m_pEditWidth</tabstop>
  <tabstop>m_pEditHeight</tabstop>
  <tabstop>m_pRadioImage</tabstop>
  <tabstop>m_pRadioPicture</tabstop>
  <tabstop>m_pButtonImage</tabstop>
  <tabstop>m_pButtonPicture</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>uiDialogFrame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>uiDialogFrame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
